users/models/identity.py

```diff
@@ -256,8 +256,8 @@ def all_absolute_profile_uris(self) -> list[str]:
         if not self.local:
             return [self.profile_uri]
         return [
-            f"https://{self.domain.uri_domain}/@{self.username}/",
             f"https://{self.domain.uri_domain}/@{self.username}@{self.domain_id}/",
+            f"https://{self.domain.uri_domain}/@{self.username}/",
```

Comment:
The ordering here is likely the source of some of the actor_uri issues for Takahe servers.